pkg/daemon/discover/discover.go

```diff
@@ -51,6 +55,7 @@ var (
 	cmName          string
 	cm              *v1.ConfigMap
 	udevEventPeriod = time.Duration(5) * time.Second
+	udevFilter      []string
```

Comment:
no need to declare this package variable. A local variable should be fine, right?

Reply:
Yeah. That should be fine since everything is inside a single function.